### ISSUE TITLE
About training loss value

### ISSUE THREAD

**Issue by lovejing0306** [ToonCrafter/ToonCrafter#46]
Hi, I am finetuning your model and training the spatial layer. Can you share your tensorboard file about the loss change?

**Comment by QingQingS** [comment]
你好，我最近也在微调tooncrafter ，请问你是微调的多大分辨率的？如果要微调高分辨率的话，VAE 的3D decoder部分是不是也要跟着调整呢

**Comment by lovejing0306** [comment]
> 你好，我最近也在微调tooncrafter ，请问你是微调的多大分辨率的？如果要微调高分辨率的话，VAE 的3D decoder部分是不是也要跟着调整呢

我微调的是512分辨率的

**Comment by Doubiiu** [comment]
The log files are inaccessible now.
